controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -258,16 +259,16 @@ func (r *StorageClusterReconciler) reconcileUninstallAnnotations(sc *ocsv1.Stora
 		err := r.Client.Get(context.TODO(), types.NamespacedName{Name: sc.Name, Namespace: sc.Namespace}, &oldSc)
 		if err != nil {
 			r.Log.Error(err, "Uninstall: Failed to get StorageCluster.", "StorageCluster", klog.KRef(sc.Namespace, sc.Name))
-			return err
+			return true, err
 		}
 		sc.ObjectMeta.ResourceVersion = oldSc.ObjectMeta.ResourceVersion
 		if err := r.Client.Update(context.TODO(), sc); err != nil {
 			r.Log.Error(err, "Uninstall: Failed to update the StorageCluster with uninstall defaults.", "StorageCluster", klog.KRef(sc.Namespace, sc.Name))
-			return err
+			return true, err
```

Comment:
This should return false as the update did not happen
```suggestion
			return false, err
```

Reply:
done